### ISSUE TITLE
Add visible labels to InnerBlocksTemplatePicker pattern selection buttons

### ISSUE THREAD

**Issue by richtabor** [WordPress/gutenberg#18737]
**Is your feature request related to a problem? Please describe.**
The lack of visual labels within the `InnerBlocksTemplatePicker` component makes it difficult to understand what the `IconButton` stands for, i.e. what pattern will be added. Currently, the only way to discover the label is to wait for the tooltip.

### **Examples:**

**1. Core Columns block:** 
The lack of labels works _ok_ here, as the icons depict exactly what pattern will be added. Apart from this particular scenario, I can't think of many instances where these icons will depict the actual pattern. 
<img width="596" alt="Screen Shot 2019-11-25 at 7 34 31 PM" src="https://user-images.githubusercontent.com/1813435/69589591-9bc60100-0fba-11ea-96e9-0b3104074685.png">

**CoBlocks Form block:** 
It's not possible to include actual interpretations of these patterns within the CoBlocks Form block. The implementation is awesome, but the lack of labels make it difficult to understand exactly what pattern will be created - unless of course you wait for the tooltip label.
<img width="595" alt="Screen Shot 2019-11-25 at 7 35 24 PM" src="https://user-images.githubusercontent.com/1813435/69589628-bef0b080-0fba-11ea-9e95-b83babdd8e62.png">

**Describe the solution you'd like**
Either there's a prop to pass visible labels, or visible labels are always present. We'd want to consider if we should use the existing `label` prop, or if we should have separate `label` and `tooltip` props.

**Describe alternatives you've considered**
Recreating the component, though that's not a consideration really.

**Comment by enriquesanchez** [comment]
Agreed this would be a a big improvement. Visible labels should always be present, they make the UI easier to understand for everyone.

**Comment by shaunandrews** [comment]
Labels could help a lot. Here's a quick mockup for what it could look like for the Core Columns block and the CoBlocks Form block:

<img width="632" alt="image" src="https://user-images.githubusercontent.com/191598/70170200-377dfe00-169a-11ea-9954-3caf5ac24b3b.png">

**Comment by gziolo** [comment]
I started #19789 to address it.
